src/ServiceFabricPubSub/SubscriberService/PackageRoot/ServiceManifest.xml

```diff
@@ -28,12 +29,8 @@
       <!-- This endpoint is used by the communication listener to obtain the port on which to 
            listen. Please note that if your service is partitioned, this port is shared with 
            replicas of different partitions that are placed in your code. -->
-      <Endpoint Name="ServiceEndpoint" />
-
-      <!-- This endpoint is used by the replicator for replicating the state of your service.
-           This endpoint is configured through a ReplicatorSettings config section in the Settings.xml
-           file under the ConfigPackage. -->
-      <Endpoint Name="ReplicatorEndpoint" />
```

Comment:
It started as a .Net Core Web API project,in order to more quickly add Kestrel and a Controller. This configuration can quickly be added in the future commit.